src/Command/Setup.php

```diff
@@ -139,6 +140,14 @@ private function getEnvName(InputInterface $input, string $cwd): EnvName
             return EnvName::get($templateName);
         }
 
+        if ($envName = $this->getEnvNameFromChoiceList($io)) {
+            try {
+                return EnvName::get($envName);
+            } catch (InvalidValueException $exception) {
+                return EnvName::CUSTOM();
```

Comment:
He uses `SymfonyStyle::choice()` which has a [default validator](https://github.com/symfony/console/blob/master/Question/ChoiceQuestion.php#L127), dunno how it behaves but maybe it re-asks naturally? Anyway re-asking seems better than going with `CUSTOM` in case of bad answer